native/explorer/src/series.rs

```diff
@@ -760,6 +760,28 @@ pub fn s_coalesce(data: ExSeries, other: ExSeries) -> Result<ExSeries, ExplorerE
     Ok(ExSeries::new(coalesced))
 }
 
+#[rustler::nif(schedule = "DirtyCpu")]
+pub fn s_in(data: ExSeries, rhs: ExSeries) -> Result<ExSeries, ExplorerError> {
+    let s: &Series = &data.resource.0;
+    let rhs: &Series = &rhs.resource.0;
+
+    let s = match s.dtype() {
+        DataType::Int8
+        | DataType::UInt8
+        | DataType::Int16
+        | DataType::UInt16
+        | DataType::Int32
+        | DataType::UInt32
+        | DataType::Int64 => s.i64()?.is_in(rhs)?,
+        DataType::Float32 | DataType::Float64 => s.f64()?.is_in(rhs)?,
```

Comment:
We only effectively use these types:

```suggestion
        DataType::UInt32 | DataType::Int64 => s.i64()?.is_in(rhs)?,
        DataType::Float64 => s.f64()?.is_in(rhs)?,
```